### PR TITLE
Central test cases

### PR DESCRIPTION
**Pull request by MariuszSkamra**

### CODE REVIEW

**Comment by MariuszSkamra** [comment]
@wopu-ot @thoh-ot FYI

**Comment by wopu-ot** [comment]
IAL test cases look good to me. I have not gotten to review the the LL test cases, but there seem to be some failures when I try to run them.

**Comment by MariuszSkamra** [comment]
@wopu-ot I hope that would help: https://github.com/EDTTool/packetcraft/pull/63

**Review by wopu-ot — APPROVED** [review]
LGTM. `ll_cis_cen_bv_05_c` should be removed in a following PR to avoid excessive rebasing work.

File: src/tests/ll_verification.py [hunk 1]
```diff
@@ -6702,120 +6737,910 @@ def iso_transmit_receive_test_mode(transport, transmitter, receiver, trace, tran
...
+    return test_sending_and_receiving_data_in_bidirectional_cis_bn_1(transport, upper_tester, lower_tester, trace)
+
+
+def ll_cis_cen_bv_48_c(transport, upper_tester, lower_tester, trace):
+    """LL/CIS/CEN/BV-48-C [Sending and Receiving Data in Bidirectional CIS, BN = 1, Encryption]"""
+    return test_sending_and_receiving_data_in_bidirectional_cis_bn_1(transport, upper_tester, lower_tester, trace,
+                                                                     ENC_KEYS)
+
+
+def ll_cis_cen_bv_05_c(transport, upper_tester, lower_tester, trace):
```
[the remark this answers is not in the document]
Comment (MariuszSkamra) [reply src/tests/ll_verification.py:6983]: agree

**Pull request merged**